Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/overview/omf-quick-start.md

   ```

   The value is indexed by a timestamp, and the numeric value that will be stored is a 32-bit floating point value.

> 2. To create the OMF type in Edge Storage, store the JSON file with the name `OmfCreateType.json` on the local device.
> 
> 3. Run the following curl command:

Reviewer: ```suggestion
1. To create the OMF type in Edge Storage, store the JSON file with the name `OmfCreateType.json` on the local device.

1. Run the following curl command:
```

   }]
   ```

   This container references the OMF type that was created earlier, and an error will occur if the type does not exist when the container is created.

> 2. To create the OMF container in Edge Storage, store the JSON file with the name `OmfCreateContainer.json` on the local device.

Reviewer: ```suggestion
1. To create the OMF container in Edge Storage, store the JSON file with the name `OmfCreateContainer.json` on the local device.
```

> 3. To create the SDS stream to store data defined by the type, run the following curl command:

Reviewer: ```suggestion
1. To create the SDS stream to store data defined by the type, run the following curl command:
```
